mmpose/models/detectors/top_down.py

```diff
@@ -225,13 +225,21 @@ def process_head(self, output, img, img_metas, return_heatmap=False):
         if 'bbox_score' in img_metas:
             score = np.array(img_metas['bbox_score']).reshape(-1)
 
-        preds, maxvals = keypoints_from_heatmaps(
-            output_heatmap,
-            c,
-            s,
-            post_process=self.test_cfg['post_process'],
-            unbiased=self.test_cfg['unbiased_decoding'],
-            kernel=self.test_cfg['modulate_kernel'])
+        if 'unbiased_decoding' in self.test_cfg:
+            preds, maxvals = keypoints_from_heatmaps(
+                output_heatmap,
+                c,
+                s,
+                post_process=self.test_cfg['post_process'],
+                unbiased=self.test_cfg['unbiased_decoding'],
+                kernel=self.test_cfg['modulate_kernel'])
+        else:
+            preds, maxvals = keypoints_from_heatmaps(
+                output_heatmap,
+                c,
+                s,
+                unbiased=self.test_cfg['unbiased_decoding'],
```

Comment:
If 'unbiased_decoding' not in self.test_cfg, do we still need this input?

Reply:
use defaults in this case

Reply:
Sorry, this is a mistake. I erroneously removed **post_process=self.test_cfg['post_process']** while I should remove **unbiased**